src/Durable/Commands/SetDurableCustomStatusCommand.cs

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+#pragma warning disable 1591 // Missing XML comment for publicly visible type or member 'member'
+
+namespace Microsoft.Azure.Functions.PowerShellWorker.Durable.Commands
+{
+    using System.Collections;
+    using System.Management.Automation;
+
+    [Cmdlet("Set", "DurableCustomStatus")]
+    public class SetDurableCustomStatusCommand : PSCmdlet
+    {
+        [Parameter(
+            Mandatory = true,
+            Position = 0,
+            ValueFromPipeline = true)]
```

Comment:
What would the use case be here for `ValueFromPipeline`? Something like: 

```powershell
# 1)
'Processing Tokyo' | Set-DurableCustomStatus

# 2)
 @{ ProgressMessage = 'Processing Seattle'; Stage = 2 }  | Set-DurableCustomStatus

```

Reply:
Yes, exactly.